### ISSUE TITLE
New Projection Methods

### ISSUE THREAD

**Issue by tedhabermann** [opengeospatial/geotiff#3]
This problem is described by Frank Warmerdam on the [Refining GeoTIFF](http://trac.osgeo.org/geotiff/wiki/RefiningGeoTIFF) page.

Since the development of the [GeoTIFF Specification](http://www.remotesensing.org/geotiff/spec/geotiffhome.html), a number of projection methods and parameters have been added. These should also be reviewed, and if they seem reasonable and in somewhat well understood and common use they should be captured in the specification.

**Comment by rouault** [comment]
One common issue is the encoding of the WebMercator projection / EPSG:3857. See https://trac.osgeo.org/gdal/ticket/5924

**Comment by rouault** [comment]
As an example of this, libgeotiff has incorporated 2 new coordinate transformation codes (that is projection methods):
```

/* Added Feb 2005 */
ValuePair(CT_CylindricalEqualArea, 28)

/* Added May 2012 - from now on we use the EPSG */
ValuePair(CT_HotineObliqueMercatorAzimuthCenter, 9815)
```

**Comment by EmDevys** [comment]
Revised 1.1 should have fixed this by use of EPSG Projected CRS codes, proposal to close this issue.

**Comment by rouault** [comment]
> Revised 1.1 should have fixed this by use of EPSG Projected CRS codes

That only partly fixes the issue. What about custom projected CRS that would want to use other projection methods than the one available in GeoTIFF ?

**Comment by EmDevys** [comment]
My wording was incomplete. should have been "by use of EPSG Projected CRS codes or by user-defined Projected CRS", also explained in Annex B.
Therefore I confirm that this issue seems to be solved (IMHO)

**Comment by cmheazel** [comment]
Addressed in version 1.1.  Closed at the GeoTIFF SWG meeting in Banff 2019
